code/game/objects/items/weapons/manuals.dm

```diff
@@ -792,6 +792,41 @@
 		</html>
 		"}
 
+// Yes I'm a madman -farie82
+var/space_laws = list("100, Damage to Station Assets" = "To deliberately damage the station or station property to a minor degree with malicious intent.",

```

Comment:
Ideally you should use the global defines in https://github.com/ParadiseSS13/Paradise/blob/master/code/__DEFINES/_globals.dm for global list.